public/js/user/home.js

```diff
@@ -17,6 +32,7 @@ formContactUs.addEventListener('submit', (e) => {
     .then((res) => {
       if (res.msg === 'done') {
         popup.style.display = 'flex';
```

Comment:
@aminalakhsham This is a very minor suggestion, but if you use classes here in the js instead of styles, you would have a little more flexibility.
For example, if you had `popup.classList.add('is-visible')` instead of `popup.style.display = 'flex'`, then later you could change the appearance of the popup without needing to update the javascript.  For example, if you decided that the popup should be display=block instead of display=flex, then you could make that change in your css instead of your js.  Even better, if you decided it should fade in with an opacity transition, you could do that easily too!

Reply:
yaa i know this way but i'm so lazy 
but first time know the js file load another time 
i think this comment very useful

Reply:
Yeah I know what you mean, and it's good to know when a piece of code is "good enough" because you have a *lot* of other things to do, instead of spending too much time on one piece of code making it perfect.
Sometimes I just keep a list to myself of things that I might fix later.  Then when you have a little time, you can look at that list and pick the worst hack that you created, and fix it.
Or for this specific example, what you wrote is totally fine for the current state of the project, and you could change it to a classlist later, if you did end up wanting to change the popup styling.  Because you might never change the popup styling.